.gitignore

```diff
@@ -48,6 +48,8 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
+tests/unit_test_data
+tests/unit_test_images
```

Comment:
Probably want to add the following too then. Although I sort of like having them show up on the `git status` so I can easily see what running a test added.
```
tests/image_check_failures/
tests/system/all_sets_results_test/
tests/system/image_check_failures/
tests/v2_3_0_all_sets/
```

Reply:
A possible workaround for your use case is to temporarily remove the `tests/` lines from your local `.gitignore` so you can see the diffs, then discard changes as needed.

I think it would be cleaner to not have the files in the diff and also eliminate the possibility of contributors pushing test outputs.